pkg/domain/stateDiffDoguConfig.go

```diff
@@ -0,0 +1,41 @@
+package domain
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-41 lines are duplicate of `pkg/domain/stateDiffGlobalConfig.go:1-42`